__tests__/app.test.js

```diff
@@ -482,6 +452,103 @@ describe('GET /api/users', () => {
     })
 })
 
+describe('QUERIES /api/reviews', () => {
+    test('200: Responds with reviews only of category queried', () => {
+        return request(app)
+        .get('/api/reviews?category=social_deduction')
+        .expect(200)
+        .then(({body}) => {
+            const {reviews} = body
```

Comment:
If someone added another review that was in the same category these tests would fail, can you expect something other than the review lengths.